projects/2023TemplateWithVite/README.md

```diff
@@ -66,5 +66,5 @@ For this template, the name of your db should be `techtonica`.
 
 ⚙️ Links that you could need:
 
-- The instructions for [pg](https://node-postgres.com/apis/pool)
-- Setup [postgres correctly](https://github.com/Techtonica/curriculum/blob/main/databases/installing-postgresql.md)
+- The instructions for [pg](https://node-postgres.com/apis/pool) (Webite)
```

Comment:
Website (s is missing)

Reply:
Thanks, Ruthie! I have corrected the spelling and the commit hash for this change is 9f223db261e90f9ae1ff69c6ae4b3edf614590cf